stack.py

```diff
@@ -28,15 +28,20 @@ def __eq__(self, other):
     'quiet': 'quiet',
     '#': 'quiet',
     '♯': 'quiet',
+
     'cond': 'cond',
-    '?': 'cond'
+    '?': 'cond',
+
+    'qcond': 'qcond',
+    '??': 'qcond', '⁇': 'qucond', # Second one is U+2047 (DOUBLE QUESTION MARK)
+    '¿': 'qcond',
```

Comment:
Perhaps format these like we format the sigils, put all the synonyms on one line.